CODE_OF_CONDUCT_en.md

```diff
@@ -1,79 +1,119 @@
+Our primary commitment is to ensure the safety and inclusion of marginalized individuals. We recognize that discussions about equity and inclusion can sometimes feel uncomfortable, but we will not prioritize the comfort of those in privileged positions over the well-being of those facing discrimination or exclusion.
 
-* Anna Carnegie, Community Manager - LSHTM
-* Diana Fajardo Pulido, Research Coordinator TRACE-LAC - Pontificia Universidad Javeriana
-* Nicolas Yanez, Clinical Data Scientist - Philips Research North America [external Committee Member]
-* Natalia Niño, Researcher - Universidad de los Andes
-* Cormekki Whitley, Chief Finance Officer - data.org
+This means that we will not take action on complaints that:
 
-You may also make a report directly to one member of the Committee by contacting them directly. If any member of the Committee has a conflict of interest with a report, they will be recused and will not have access to the content or process of the report follow-up, which will be handled by the other members of the Committee.
+* Claim so-called "reverse" discrimination (e.g., “reverse racism” or “reverse sexism”), as systemic inequalities do not operate in reverse.  
+* Object to reasonable personal boundaries set by community members, such as requests to end a conversation or disengage.  
+* Focus on the tone of someone’s communication rather than the substance of their message.  
+* Criticize or challenge someone for speaking up about discrimination, exclusion, or oppressive behavior.
 
-Upon submitting your report, you will receive a copy of your responses via email. You should receive a separate acknowledgement of receipt via email from the Code of Conduct Committee within 48 hours on weekdays, or within 72 hours for an event that takes place on a weekend.
+## Reporting and enforcement
+
+The Epiverse Code of Conduct Committee includes Diana Fajardo Pulido (Epiverse Community Engagement and Involvement Manager), Karim Mané (Epiverse Research Software Engineer), Sebastian Funk (Epiverse lead at LSHTM) <!-- and xxx (External Committee Member). -->
+
+### How to report a violation
+
+If you experience or witness behavior that violates this Code of Conduct, or if you have concerns about someone’s safety, you can report it through:  
+**Email:** Send a report to [conduct@epiverse.org](mailto:conduct@epiverse.org)   
+**In-person Events:** Contact any member of the Code of Conduct Committee or designated event moderators. 
+<!-- **Online Form:** Submit a confidential report \[link to form\]. --> 
+Reports can be submitted anonymously or with identifying details. All reports will be treated confidentially and with care for the reporter’s safety. Upon submitting your report, you will receive a copy of your responses via email. You should receive a separate acknowledgement of receipt via email from the Code of Conduct Committee within 48 hours on weekdays, or within 72 hours for an event that takes place on a weekend.
```

Comment:
Is this currently possible given that the report will be made to an email from an email? Maybe rather use a form that allows the user to not send their email?